src/Administration/Resources/app/administration/src/module/sw-mail-template/page/sw-mail-template-detail/index.js

```diff
@@ -38,7 +38,8 @@ export default {
             mailTemplate: false,
             testerMail: '',
             mailTemplateId: null,
-            mailPreview: null,
```

Comment:
Removing this is a breaking change

Reply:
Good morning @seggewiss! Does renaming the html part help? Because changing the structure of the response is a bc as well imho?

Reply:
For me this is fine. We just can't remove the property and would need to deprecate like here:
https://github.com/shopware/shopware/blob/trunk/src/Administration/Resources/app/administration/src/app/asyncComponent/media/sw-media-modal-folder-settings/index.js#L42